serpent-moltres/full-core/postprocessing.py

```diff
@@ -5,7 +5,66 @@
 from matplotlib.collections import PatchCollection
 from matplotlib.pyplot import gca
 import pathmagic
-from auxiliary import plot_serpent_axial_collapse
+import pandas as pd
```

Comment:
Rather than repeat my comments from other PRs.... 


![image](https://user-images.githubusercontent.com/393899/105382258-7db35b00-5bd5-11eb-9aca-d9caf8faecf1.png)